syft/pkg/cataloger/golang/scan_bin.go

```diff
@@ -41,7 +41,11 @@ func scanFile(reader unionReader, filename string) ([]*debug.BuildInfo, []string
 			}
 			// in this case we could not read the or parse the file, but not explicitly because it is not a
 			// go-compiled binary (though it still might be).
-			log.Warnf("golang cataloger: failed to read buildinfo (file=%q): %v", filename, err)
+			// TODO: We should change this back to "warn" eventually.
+			//  But right now it's catching too many cases where the reader IS NOT a Go binary at all.
+			//  It'd be great to see how we can get those cases to be detected and handled above before we get to
+			//  this point in execution.
+			log.Infof("golang cataloger: unable to read buildinfo (file=%q): %v", filename, err)
```

Comment:
@luhring just so I'm clear - you still think this should show up in its current state when a user passes `-v`?

Any consideration for going down to debug for `-vv` because of how many cases are being caught?

Reply:
That's a good question. I think this is a notable occurrence, so my personal opinion would be to have it show up w/ `-v`. I also think that ideally we move this back to `WARN` (so it shows up even without `-v`) — but only once we've solved the issue if this conditional branch being hit more correctly and less often.

I don't have a strong opinion on this, though. I see this as a temporary solution. I defer to you on this!